lisa/utils.py

```diff
@@ -217,6 +218,33 @@ def _yaml_include_constructor(cls, loader, node):
         with open(path, 'r', encoding=cls.YAML_ENCODING) as f:
             return cls._yaml.load(f)
 
+    @classmethod
+    def _yaml_env_var_constructor(cls, loader, suffix, node):
+        """
+        Provide a !include tag in YAML that can be used to include the content
+        of an environment variable, and converting it to a Python type:
+
+        !env:int MY_ENV_VAR
```

Comment:
I'd like the docstring to mention CI/Jenkins, so people like me don't start thinking they *need* to use env variables to setup TestEnv's.

Reply:
Such documentation does not really belong to here. You also don't *need* !include, !var and !call , they are just features of the configuration "on top" of YAML. This class is used as the base class for basically anything that would use a config file. Maybe we can have a non-docstring documentation on how to do an advanced setup, with examples of these tags to show people what they can use to fit their specific use case (like the fact that you can put all these configurations into one file).

Reply:
Having a few lines of docstring explaining *why* this feature is there when it's not obvious (to me) at a first glance can never hurt.